CHANGES.md

```diff
@@ -1,6 +1,15 @@
 Change Log
 ==========
 
+### 1.46 - 2018-05-01
+
+##### Breaking Changes :mega:
+* `Camera.distanceToBoundingSphere` now returns the signed distance to the bounding sphere. Positive values indicate that the bounding sphere is in the positive half-plane of the camera position and view direction while a negative value indicates it is in the negative half-plane.
+
+##### Additions :tada:
+* Added option `logDepthBuffer` to `Viewer`. With this option there is typically a single frustum using logarithmic depth rendered. This increases performance by issuing less draw calls to the GPU and helps to avoid artifacts on the connection of two frustums. [#5851](https://github.com/AnalyticalGraphicsInc/cesium/pull/5851)
```

Comment:
Did we also change the default near value?

Reply:
Late to the party, but should this be `logarithmicDepthBuffer` in `Scene`?

Reply:
https://github.com/AnalyticalGraphicsInc/cesium/pull/6465